ci/nightly/pipeline.template.yml

```diff
@@ -101,6 +101,28 @@ steps:
               - common-ancestor
               - --max-concurrency
               - "256"
+    - id: connection-initiation-benchmark
+      label: "Connection initiation benchmark against merge base or 'latest'"
+      timeout_in_minutes: 60
+      agents:
+        queue: linux-aarch64-large
+      artifact_paths: junit_*.xml
+      plugins:
+        - ./ci/plugins/mzcompose:
+            composition: scalability
+            args:
+              - --target
+              - HEAD
+              - --target
+              - common-ancestor
+              - --regression-against
+              - common-ancestor
+              - --min-concurrency
+              - "256"
```

Comment:
I think you still need to benchmark concurrency=1 here . What if connection establishment started taking 1 second individually, but be 100% scalable? We would want Buildkite to detect this and fail.

So maybe just set --min-concurrency to 1 and `--exponent-base` to `4` or `8` ?

Reply:
✅